Libraries/dnYara.Interop/Interops/Types/YR_RULE.cs

```diff
@@ -3,34 +3,38 @@
 
 namespace dnYara.Interop
 {
-    [StructLayout(LayoutKind.Sequential)]
-    public struct YR_RULE
+    [StructLayout(LayoutKind.Explicit, Size = 448)]
+    public unsafe struct YR_RULE
```

Comment:
this and YR_RULES needed explicit layout in the end, and the code from CppSharp required the use of `fixed`, which then required the `unsafe` keyword.  This might be able to be reworked a bit to not require the specific field offsets, but I spent a few hours and couldn't get it so IMO the Explicit LayoutKind will need to stay.

Reply:
I just checked the previous YR_RULE struct size, it has a size of 448 (old version) and seems the same than the new unsafe struct. Can you check on your side by rolling back only this struct ?

Reply:
Yep, that worked.